experimenter/experimenter/theme/static_src/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "theme",
+  "version": "3.6.0",
+  "description": "",
+  "scripts": {
+    "start": "npm run dev",
+    "build": "npm run build:clean && npm run build:tailwind",
+    "build:clean": "rimraf ../static/css/dist",
+    "build:tailwind": "cross-env NODE_ENV=production tailwindcss --postcss -i ./src/styles.css -o ../static/css/dist/styles.css --minify",
+    "dev": "cross-env NODE_ENV=development tailwindcss --postcss -i ./src/styles.css -o ../static/css/dist/styles.css -w",
+    "tailwindcss": "node ./node_modules/tailwindcss/lib/cli.js"
```

Comment:
scripts are able to find binaries that are installed to `node_modules/.bin`, so you should be able to do:
```
"tailwindcss": "tailwindcss"
```